tmp/FAQ.md

```diff
@@ -0,0 +1,61 @@
+# FAQ
```

Comment:
To chyba skomitowaliście przez przypadek. Polecam dodanie katalogu tmp do .gitignore.